kolibri/plugins/learn/assets/src/views/LearningActivityBar.vue

```diff
@@ -66,6 +66,8 @@
             </div>
           </template>
         </CoreMenu>
+        <DeviceConnectionStatus :deviceId="deviceId" />
```

Comment:
Non-blocking, but just double checking you intentionally left out the color prop here :)